### DISCUSSION TITLE
please help me create a website like the link i sent. i find it very difficult to create a website like that and please help!

### DISCUSSION THREAD

**Discussion by luuthienviet** [tailwindlabs/tailwindcss#15441] [Help]
https://coffeestyle-template.webflow.io/

**Comment by sharathdaniel** [comment]
Either learn to do it or find someone who can do it. But nobody will do the entire site for free.

**Comment by tembbo** [comment]
Hey! While the forum is great for technical questions, this kind of project request would be better of on freelance platforms like Upwork or Fiverr. That said, if you have specific questions about implementation (related to tailwind!), feel free to ask those here!
